data/dev/analysis/ota/ota002/rubrics/gbw_one_stage.json

```diff
@@ -1,43 +1,77 @@
 {
   "rubric_id": "gbw_one_stage",
   "version": "0.1.0",
-  "weights": { "correctness": 0.45, "completeness": 0.25, "groundedness": 0.20, "safety": 0.10 },
+  "weights": {
+    "correctness": 0.45,
+    "completeness": 0.25,
+    "groundedness": 0.2,
+    "safety": 0.1
+  },
   "criteria": [
     {
       "id": "topology",
       "desc": "Identifies a fully differential telescopic/folded cascode one-stage OTA in the Topology section.",
       "required": true,
       "section": "Topology",
-      "patterns_any": ["telescopic", "folded cascode", "differential", "fully differential"],
+      "patterns_any": [
+        "telescopic",
+        "folded cascode",
+        "differential",
+        "fully differential"
+      ],
       "min_any": 2,
-      "anti_patterns": ["two-stage", "miller compensation", "second stage", "single-ended"],
-      "weight": 0.10
+      "anti_patterns": [
+        "two-stage",
+        "miller compensation",
+        "second stage",
+        "single-ended"
+      ],
+      "weight": 0.1
     },
     {
       "id": "key_relation",
-      "desc": "States GBW ≈ gm/CL for one-stage OTA and conditions for validity.",
+      "desc": "States GBW \u2248 gm/CL for one-stage OTA and conditions for validity.",
       "required": true,
       "patterns_any": [
         "gm/CL",
-        "GBW ≈ gm/CL",
-        "UGF ≈ gm/CL",
-        "GBW = gm / (2 * π * CL)"
+        "GBW \u2248 gm/CL",
+        "UGF \u2248 gm/CL",
+        "GBW = gm / (2 * \u03c0 * CL)"
+      ],
+      "anti_patterns": [
+        "ro*C_L sets GBW",
+        "CL/gm"
       ],
-      "anti_patterns": ["ro*C_L sets GBW", "CL/gm"],
       "weight": 0.25
     },
+    {
+      "id": "device_identification",
+      "desc": "Identifies which device provides gm for GBW (input transistor).",
+      "patterns_any": [
+        "gm_M",
+        "M1",
+        "M2",
+        "input",
+        "gm of M"
+      ],
+      "min_any": 1,
+      "weight": 0.05
+    },
```

Comment:
Device identification is not a valid question to ask for Cascode modality. You could potentially multiplex the rubric for when `modality=cascode`  correctly identifies "gm of the InputPair motif"